Make the section describing spanning trees non-normative (#199)
ircdocs/modern-irc: _includes/concepts.md

Communication on a one-to-one basis is usually only performed by clients, since most server-server traffic is not a result of servers talking only to each other.

> Servers are be able to send a message from any one client to any other. Outside of networking issues, all servers are always able to send a message in exactly one direction along the spanning tree to reach any client. Thus the path of a message being delivered is the shortest path between any two points on the spanning tree.

Reviewer: "are be able"?

> Servers are be able to send a message from any one client to any other. Outside of networking issues, all servers are always able to send a message in exactly one direction along the spanning tree to reach any client. Thus the path of a message being delivered is the shortest path between any two points on the spanning tree.

Reviewer 2: ```suggestion
Servers should be able to send a message from any one client to any other. Servers send a message in exactly one direction along the spanning tree to reach any client. Thus the path of a message being delivered is the shortest path between any two points on the spanning tree.
```
Just simplifies things and focuses on the conceptual description – no need to mention networking issues here because that won't affect the way messages are passed along the way conceptually.
